content/the-ember-times-issue-201.md

```diff
@@ -0,0 +1,173 @@
+---
+title: The Ember Times - Issue No. 201
+authors:
+  - anne-greeth-schot-van-herwijnen
+  - NullVoxPopuli
+date: 2022-06-27T00:00:00.000Z
+tags:
+  - newsletter
+  - '2022'
+---
+
+👋 Emberistas! 🐹
+
+Embroider is legit.
+
+<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+
+---
+
+## [🧵 1. Embroider is legit](https://github.com/embroider-build/embroider)
+
+Embroider has been _declared_ stable since v1.0.0, or around 6 months ago (released mid January 2021). Embroider is the 2nd generation build pipeline to Ember Apps that enables the usage of “Standard JavaScript” and leverages standard JavaScript packagers such as Webpack.
+
+Since even before v1.0.0, addon-authors have been adding **two** embroider-centric tests in C.I. for both "maximum compatibility" and "maximum strictness" to their ember-try configs.
+`ember-cli` has even added an `--embroider` flag out of the box for starting new projects with embroider.
+Try embroider out today in a new app!
+
+```bash
+ember new my-app --embroider
+```
+
+Or, add embroider to an existing app via this codemod:
+
+```bash
+npx ember-apply embroider
+```
+
+This [`ember-apply`](https://github.com/NullVoxPopuli/ember-apply/) command automates the setup instructions in the [embroider README](https://github.com/embroider-build/embroider#how-to-try-it).
+
+And speaking of the README, it has a section of if _you_ should use embroider or not, today.
+
+> Several large, heavily-tested Ember apps are shipping to production with Embroider. So if you are excited to adopt Embroider, it is a reasonable choice. The main risks to be aware of if you choose to use Embroider in production are:
+>
+> - you're likely to discover some Ember addons don't work or break your build
+> - Embroider's own configuration options are subject to change, so you'll need to read the CHANGELOG.md when updating the Embroider packages.
+>
+> Alternatively, it is totally safe to stick with the traditional build pipeline and wait for the official cutover point when EmberCLI starts generating new apps with Embroider by default.
+Note that while some addons may break your build, this may be for the greater good. For example, instead of relying on ember-specific solutions to certain build-time behaviors, you may now follow heavily documented, and asked about, and iterated on Webpack plugins. Additionally, removing addons with build-time behavior will speed up your overall dev-server start time.
```

Comment:
It looks like this paragraph is inheriting the block-quote from above, so it reads as if it's part of the quote from the `README.md`. I think maybe inserting a line between will fix it.